asic/testbench/testbench.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright 2023 Sycuricon Group
```

Comment:
Could you please clarify whether this file is intended for use with Verilator or if it is a dangled unused file?
BTW, you should also remove the `asic2` directory and merge the `verilate` directory to the `asic` directory.